src/Network/Email/AbstractTransport.php

```diff
@@ -1,3 +1,16 @@
 <?php
-// @deprecated Backward compatibility with 2.x, 3.0.x
-class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+// @codingStandardsIgnoreFile
+namespace Cake\Network\Email;
+
+class_alias(\Cake\Mailer\AbstractTransport::class, AbstractTransport::class);
+
+if (class_exists(AbstractTransport::class)) {
+    return;
+}
```

Comment:
@thinkingmedia 

> It's only read by the IDE. When PHP executes it never reaches the class declaration after the if(class_exists()). 

I don't think so. That will be executed if the class has no interfaces and no traits. This behavior is documented in the PHP manual.
See http://php.net/manual/en/migration5.incompatible.php#example-6460